### ISSUE TITLE
Allow a repository administrator to make changes to the 'Simple item page'  from the GUI

### ISSUE THREAD

**Issue by bjpot** [DSpace/DSpace#8805]
In our present iteration of XMLUI DSpace, modifying the 'Simple item page' fields requires understanding intricate alterations in the DSpace installation directory on the server, or contracting with a third-party.

A potential enhancement for DSpace 7 Angular could be to introduce a functionality that facilitates simpler modification of the 'Simple item page' by the repository administrator through the user interface:

- Assign field labels and link them to specific Dublin Core fields.
- Rearrange the sequence of fields.
- Amend existing labels and fields.
- Enable selected fields to initiate actions, such as making a 'DOI' field clickable and resolving to an external URI in a new tab.

**Comment by tdonohue** [comment]
Agreed, this would be a nice change.  However, this may be complex to implement as different "Simple Item Page" settings are possible for different Entity Types (see [Configurable Entities](https://wiki.lyrasis.org/display/DSDOC7x/Configurable+Entities)).  It is also technically possible to configure a different "Simple Item Page" per Theme.  So, it may be that each Item needs to hold its own settings for which fields are displayed on its "Simple Item Page", or similar.

In the meantime, it is possible to modify the "Simple Item Page" from the theme itself by modifying the HTML directly (in your theme).  For instance...
* To modify the "Simple Item Page" for a normal Item, modify [`untyped-item.component.html`](https://github.com/DSpace/dspace-angular/blob/dspace-7.5/src/app/item-page/simple/item-types/untyped-item/untyped-item.component.html) (or pull it into your theme and change it)
* To modify the "Simple Item Page" for a Publication Entity type, modify [`publication.component.html`](https://github.com/DSpace/dspace-angular/blob/dspace-7.5/src/app/item-page/simple/item-types/publication/publication.component.html)
* etc...

Requires a volunteer to investigate ways to achieve this.  I agree that conceptually this is a nice idea, but it needs scoping, analysis.
